packages/cli/src/util/createServer.ts

```diff
@@ -95,10 +101,25 @@ function pipeOutputToSignale(stream: Readable) {
   });
 }
 
-export type CreatePrismOptions = {
+function isProxyServerOptions(options: CreateBaseServerOptions): options is CreateProxyServerOptions {
+  return !options.dynamic && 'upstream' in options;
+}
+
+type CreateBaseServerOptions = {
   dynamic: boolean;
   cors: boolean;
-  host?: string;
+  host: string;
   port: number;
   operations: IHttpOperation[];
+  multiprocess: boolean;
+  log: 'stdout' | 'httpResponse' | 'httpHeader';
 };
+
+export interface CreateProxyServerOptions extends CreateBaseServerOptions {
+  dynamic: false;
```

Comment:
`dynamic` makes sense only in mock command, right? If so, let move it there.

Reply:
28ebfc2